components/dashboard/src/hooks/use-query-param.ts

```diff
@@ -0,0 +1,14 @@
+/**
```

Comment:
FWIW this case is good here as it's a generic utility. The file name should be plural, though.